docs/deployment-integration.md

```diff
@@ -9,17 +9,24 @@ reports misbehavior for slashing. This is the core and only
 contract that a rollup deploys to become a BSN. The contract must be deployed
 on Babylon Genesis as it relies on direct interaction with it.
 
-## Governance Notes
-<img width="3100" height="772" alt="governance" src="./assets/governance.png" />
+## Overview
+
+The BSN deployment process consists of four main steps:
 
+1. **Upload Contract Code**: Upload the compiled WASM bytecode to Babylon Genesis to obtain a code ID
+2. **Instantiate Contract**: Create a contract instance using the code ID with your specific BSN configuration 
+3. **Register BSN**: Register your BSN with Babylon using the contract address to enable finality provider participation
+4. **Maintain Contract**: Ongoing contract management and operational tasks as needed
```

Comment:
Would it be more clear if the rest of the sections follow the order here?

Reply:
I think it already does? In this document we agreed not to cover the basic of uploading and isntantiating contract in cosmwasm as this this is not cosmwasm doc. 

But in this document we have
1. Instantiation
2. Registration
3. Contract maintaince

in this order